src/databricks/labs/ucx/hive_metastore/locations.py

```diff
@@ -212,6 +231,9 @@ def resolve_mount(self, location: str | None) -> str | None:
         if not location.startswith('dbfs:'):
             return location  # not a mount, save some cycles
         for mount in self._mounts_snapshot:
+            # Skip the root mount
```

Comment:
Please rewrite this WHAT comment into a WHY comment

Reply:
No longer needed